package.json

```diff
@@ -310,14 +318,7 @@
       "typings"
     ],
     "nohoist": [
-      "@fluentui/react-northstar-fela-renderer/stylis",
```

Comment:
Is it expected that `stylis` entries are removed?

Reply:
yes, that no hoist is not needed. yarn properly allocates those packages.

- stylis 3 lives in fela-renderer node_modules 
  - has no deps which could cause issues when hoisted so all good 
- stylis 4 is hoisted to root node_modules ( used by emotion-renderer ) 
  -  has no deps which could cause issues when hoisted so all good

Reply:
AFAIR there was a problem before with typings: `stylis@3` has them built-in, `stylis@4` relies on `@types/stylis`. But it seems that it's not longer an issue as build is passing. May be because Griffel is no longer in this repo.